README.md

```diff
@@ -228,236 +250,52 @@ messages about the conversion.
 
 ## Example Usage
 
-The following examples assume ``harbourbridge`` has been added to your PATH
-environment variable.
-
-To use HarbourBridge on a PostgreSQL database called mydb, run:
-
-```sh
-pg_dump mydb | harbourbridge
-```
-
-The tool can also be applied to an existing pg_dump file:
-
-```sh
-harbourbridge < my_pg_dump_file
-```
-
-To specify a particular Spanner instance to use, run:
-
-```sh
-pg_dump mydb | harbourbridge -instance my-spanner-instance
-```
-
-By default, HarbourBridge will generate a new Spanner database name to populate.
-You can override this and specify the database name to use by:
+Driver types (source DB) that are supported by Harbourbridge:
 
-```sh
-pg_dump mydb | harbourbridge -dbname my-spanner-database-name
-```
+1. mysqldump
+2. pgdump
+3. mysql
+4. postgres
 
-HarbourBridge generates a report file, a schema file, and a bad-data file (if
-there are bad-data rows). You can control where these files are written by
-specifying a file prefix. For example,
-
-```sh
-pg_dump mydb | harbourbridge -prefix mydb.
-```
+Please refer below mentioned links for example usage of MySQL and PostgreSQL database:
```

Comment:
Nits: Same here. We always talk about PostgresSQL/MySQL. Suddenly, the order has changed. I would prefer the same order, `PostgresSQL` and then `MySQL`.